ConversationsApp/Models/CoreData/ChannelTypeDataMigration.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  ChannelTypeDataMigration.swift
+//  ConversationsApp
+//
+//  Created by Alejandro Orozco Builes on 27/11/23.
+//  Copyright © 2023 Twilio, Inc. All rights reserved.
+//
+
+import CoreData
+
+
+class ChannelTypeDataMigration: NSEntityMigrationPolicy {
+    override func createDestinationInstances(
+        forSource sInstance: NSManagedObject,
+        in mapping: NSEntityMapping,
+        manager: NSMigrationManager
+    ) throws {
+
+        guard sInstance.entity.name == "PersistentParticipantDataItem" else {
+            return
+        }
+        
+        let previousChannelType = sInstance.primitiveValue(forKey: "type") as? Int16 ?? 0
+
+        let channel: String?
+
+        switch Int(previousChannelType) {
+            case 0:
+                channel = nil
+            case 1:
+                channel = "other"
+            case 2:
+                channel = "chat"
+            case 3:
```

Comment:
As I see from [this screenshot](https://private-user-images.githubusercontent.com/4898461/286286439-3428daa0-5ebe-47af-8bdf-31db65c59949.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTc1ODkxOTAsIm5iZiI6MTcxNzU4ODg5MCwicGF0aCI6Ii80ODk4NDYxLzI4NjI4NjQzOS0zNDI4ZGFhMC01ZWJlLTQ3YWYtOGJkZi0zMWRiNjVjNTk5NDkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDYwNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA2MDVUMTIwMTMwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDE0MGRiMjVlZWQ2NWNjODk2ZmUzZTM4YWY1OTdkY2I2OWE0OWRhZTU4N2JhYmZhNjc5OTFkNjQ4NjgxMDEzYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.IhM_OB0qFUg6jhBNM01iWwHrvvheIY0l-GYSSDD_ifA) is should be like:
```
case 3:
   channel = "sms"
case 4:
   channel = "whatsapp"
```
isn't it?

Reply:
I think you're right, added both. Thank you 👍